asmcomp/linearize.mli

```diff
@@ -36,8 +36,8 @@ and instruction_desc =
   | Lcondbranch of Mach.test * label
   | Lcondbranch3 of label option * label option * label option
   | Lswitch of label array
-  | Lsetuptrap of label
-  | Lpushtrap
+  | Lentertrap
+  | Lpushtrap of {lbl_handler:label}
```

Comment:
Here and elsewhere, I think the usual convention is to have spaces inside the curly brackets and around the colon.  I also recommend a trailing semicolon to ease refactoring.  So: `{ lbl_handler : label; }`.